packages/cli/commands/feedback.js

```diff
@@ -0,0 +1,34 @@
+const open = require('open');
+
+const { i18n } = require('@hubspot/cli-lib/lib/lang');
+const {
+  FEEDBACK_OPTIONS,
+  FEEDBACK_URLS,
+} = require('@hubspot/cli-lib/lib/constants');
+const { logger } = require('@hubspot/cli-lib/logger');
+
+const {
+  feedbackTypePrompt,
+  bugPrompt,
+  generalPrompt,
+} = require('../lib/prompts/feedbackPrompt');
+
+const i18nKey = 'cli.commands.project.subcommands.feedback';
+
+exports.command = 'feedback';
+exports.describe = i18n(`${i18nKey}.describe`);
+
+exports.handler = async () => {
+  const { type } = await feedbackTypePrompt();
+
+  const promptForType =
+    type === FEEDBACK_OPTIONS.BUG ? bugPrompt : generalPrompt;
+  const { shouldOpen } = await promptForType();
+
+  if (shouldOpen === 'Y') {
```

Comment:
Rather than check for "Y" or "N", I believe you can make the shouldOpen prompts `type: confirm`, which should return a boolean. ([docs](https://www.npmjs.com/package/inquirer))